docs/tools/README.md

```diff
@@ -1,4 +1,11 @@
 ---
 ---
 
-# Tools Hub
+## Tools Hub
+
+<Tools />
+
+
+---
+
+These tools have been developed by various teams contributing to the Balancer Ecosystem or wider DeFi through grants, as general contributors or separate platforms entirely. This hub serves as an open source repo of these tools created to help users and devs navigate, analyse and interact with the Balancer Ecosystem. This is continously growing so if you have something you think should be on this page, get in touch and add your project to the [open source repo here](https://github.com/balancer/docs/tree/main/docs/toolshub) for it to be added! 
```

Comment:
Link is broken, should go do `docs/tools`

Reply:
fixed